src/main/java/com/jfrog/ide/idea/scan/SourceCodeScannerManager.java

```diff
@@ -113,7 +126,11 @@ public void run(@NotNull com.intellij.openapi.progress.ProgressIndicator indicat
                     log.info("Advanced source code scan is already in progress");
                     return;
                 }
-                sourceCodeScanAndUpdate(new ProgressIndicatorImpl(indicator), ProgressManager::checkCanceled, log);
+                try {
+                    sourceCodeScanAndUpdate(new ProgressIndicatorImpl(indicator), ProgressManager::checkCanceled, log);
+                } catch (IOException e) {
+                    logError(Logger.getInstance(), "Failed to run advanced source code scanning.", e, true);
+                }
```

Comment:
This change is not necessary - sourceCodeScanAndUpdate shouldn't return an exception

Reply:
It is necessary. The new flow of reading the new `JFrogApplicationsConfig` may throw an IO Exception.